flowbench/BenchNet2.actor.cpp

```diff
@@ -61,3 +62,31 @@ static void bench_net2(benchmark::State& benchState) {
 }
 
 BENCHMARK(bench_net2)->Range(1, 1 << 16)->ReportAggregatesOnly(true);
+
+ACTOR static Future<Void> benchDelayActor(benchmark::State* benchState, bool useYield) {
+	// Number of random delays to start to just to populate the run loop
+	// priority queue
+	state int64_t timerCount = benchState->range(0);
+	state std::vector<Future<Void>> futures;
+	state DeterministicRandom rand(platform::getRandomSeed());
+	while (--timerCount > 0) {
+		futures.push_back(delay(1.0 + rand.random01(), getRandomTaskPriority(rand)));
+	}
+
+	while (benchState->KeepRunning()) {
+		wait(useYield ? yield() : delay(0));
+	}
+	benchState->SetItemsProcessed(static_cast<long>(benchState->iterations() + timerCount));
```

Comment:
I think `timerCount` should not be included here, since these tasks are not processed during the benchmarked section.

Reply:
Good call, done.

Reply:
This did affect the results, I updated the description.